infrastructure/identity/ecr.tf

```diff
@@ -0,0 +1,7 @@
+resource "aws_ecr_repository" "identity-admin_webapp" {
```

Comment:
I have duplicated the ECR repo into both experience/identity stacks for now - i'm not sure where we want to push that to eventually, but deleting the experience stack version might bork parallel work by @jamesgorrie to fix that build.